src/clj_kondo/impl/core.clj

```diff
@@ -321,7 +321,7 @@
 
 ;;;; threadpool
 
-(defn lint-task [ctx ^java.util.concurrent.LinkedBlockingDeque deque dev?]
+(defn analyze-task [ctx ^java.util.concurrent.LinkedBlockingDeque deque dev?]
```

Comment:
This series of renames reflects the fact that these tools call `ana/analyze-input`, not one of the linter namespaces. They do, however, lint as a side effect of calling `ana/analyze-input`. Would `analyze-and-lint-task` be better?

Reply:
It's fine with me this way.